### ISSUE TITLE
Support of Background-Only images inside dataset

### ISSUE THREAD

**Issue by MyVanitar** [weiliu89/caffe#715]
@weiliu89 

Hello,

Does the SSD support background-only images inside training dataset?
if Yes, How the corresponding `.xml` file should look like?

**Comment by zoooo0820** [comment]
@VanitarNordic 
Hello,
I found a lot of discussions about the training ssd with background-only images. But there seems to be no conclusion. Have you solved this problem?

**Comment by MyVanitar** [comment]
@zoooo0820 

No, for this purpose the code must be changed which the author has not performed yet

**Comment by zoooo0820** [comment]
@VanitarNordic 
OK, thanks anyway.

**Comment by xhzzhang** [comment]
Hello, i have same issue, anybody can help me to fix it? thanks.

**Comment by MyVanitar** [comment]
@xhzzhang 

The author does not care about this
